mindsdb/integrations/handlers/byom_handler/proc_wrapper.py

```diff
@@ -1,6 +1,16 @@
-import os, sys
-import pickle
+"""
+Utility functions used in the 'Bring Your Own Model' (BYOM) engine.
+
+These functions interact with:
+    - interfaces (stdin, stdout),
+    - python files,
+    - and the actual BYOM engine.
+"""
+
```

Comment:
Doc to add/replace:
Wraps and run python code in separate python proceess. 
Communicates with parent process throughout stdin/out using pickle to serialize objects.
Flow:
 - Receives module code, method with params and stored attributed from parrent process
 - Creates python class object from code
 - Creates instance from class and filled with stored attributes
 - Calls choosen method of class with params
 - Packes response and sends to stdout
 - Exit